Cargo.toml

```diff
@@ -18,4 +18,4 @@ rustyline = "2"
 regex = "1"
 
 [dev-dependencies]
-tempdir = "^0.3.7"
\ No newline at end of file
+tempdir = "^0.3.7"
```

Comment:
[FYI (it is not issue about this PR)]`tempdir` has been deprecated and [`tempfile`](https://github.com/Stebalien/tempfile) is now recommended.